server/boot/user.js

```diff
@@ -266,6 +272,42 @@ module.exports = function(app) {
     return res.redirect('/' + username);
   }
 
+  function getUnlinkSocial(req, res, next) {
```

Comment:
thanks, I'm not aware about userIdentity.
I assume what should I do is define `const UserIdentity = app.models.UserIdentity;` then call `UserIdentity.destroyAll` targeting specific `userId` and `provider` ? It's "am" (GMT +7) in here, haven't had a chance to fix the commit, will do it tomorrow my time.